rmgpy/data/thermo.py

```diff
@@ -1456,7 +1517,7 @@ def __addPolycyclicCorrectionThermoData(self, thermoData, molecule, polyring):
         # polycylic ring in molecule and match group)
         # otherwise, apply heuristic algorithm
         if not isPolyringPartialMatched(polyring, matched_group):
-            thermoData = addThermoData(thermoData, matched_group_thermodata, groupAdditivity=True)
+            thermoData = addThermoData(thermoData, matched_group_thermodata, groupAdditivity=True, verbose=True)
```

Comment:
Here we want to turn on the comments of polycyclic correction to pass the unittest. 
Typically this comment is very short and also very helpful to check if the ring correction is calculated correctly. 
So I think probably here we could leave it on.